src/components/learner-credit-management/data/constants.js

```diff
@@ -22,6 +22,12 @@ export const API_FIELDS_BY_TABLE_COLUMN_ACCESSOR = {
   courseListPrice: 'course_list_price',
 };
 
+// Course pace text
```

Comment:
Thank you for making sure we stick to a single `constants` file within this module 🙌